index.html

```diff
@@ -202,6 +202,12 @@ <h1>The Registration Process</h1>
 description of the addition.
       </li>
       <li>
+The name of each extension MUST be indicative of its function, or &mdash; as with
```

Comment:
I like the statement "Avoid generic property names such as "myProperty" and DID Method names such as "identity" or "registry"." 

However, I think a name being indicative of it's function is inherently subjective and relative to the usage of language at the time. As an example, what did "indy" mean before the ledger was created? What about 3 or "google"? Originally, Google was a term used to indicate a very large number where as now it has a clear brand association to it due to the brand they've built around it.

So let me present some examples of some edge cases I can think up that may not align well with this:

1. `did:gov` -> registered by a single government
2. `did:abc` -> registered by Alphabet holdings (google)
3. `did:6b88c` -> registered by a bitcoin method because it's the first 5 character of a hex encoded SHA-1 hash of the term "bitcoin"
4. `did:vdr` -> a generic term for a method which encapsulates a variety of other verifiable data registries into a single method

Reply:
Yup. This is much more complex than it seems, and any simple guideline will quickly be run over by clever people, not too different than domain squatters who grab a bunch of one-char-different domains resembling some popular domain, and filling them with monetized pages. Unfortunately, I think this is true for both the "avoid generic" and "indicative of function" guidelines.